services/galley/src/Galley/API/Util.hs

```diff
@@ -113,6 +113,11 @@ acceptOne2One usr conv conn = case Data.convType conv of
                  $ "Connect conversation with more than 2 members: "
                 <> LT.pack (show cid)
 
+isOnlyFullPermissionMember :: Foldable m => UserId -> m TeamMember -> Bool
+isOnlyFullPermissionMember u =  not . any otherHasFullPerms
+  where
+    otherHasFullPerms x = hasFullPermissions x && x^.userId /= u
```

Comment:
The same code is introduced in brig [here](https://github.com/wireapp/wire-server/pull/56/files#diff-2aa0b2b96df0489f4edaa57c8c56012bR604). Refactor into the `galley-types` package, next to `hasFullPermissions` ?

Reply:
👍